rllib/core/learner/tf/tf_learner.py

```diff
@@ -267,6 +267,25 @@ def _load_optimizers(self, path: Union[str, pathlib.Path]) -> None:
     def set_weights(self, weights: Mapping[str, Any]) -> None:
         self._module.set_state(weights)
 
+    @override(Learner)
+    def get_optimizer_weights(self) -> Mapping[str, Any]:
```

Comment:
I'm trying to find a way to reuse these functions when saving the optimizer state, but its difficult since there is actually little overlap -- when saving the optimizer state, we actually save in native tensorflow format instead of numpy.